### ISSUE TITLE
Content Security Policy Header bug - home pages with ads - Safari

### ISSUE THREAD

**Issue by sareh** [bbc/simorgh#7001]
**Describe the bug**
When visiting a home page with ads on Test, in Safari, there are content security policy header errors.

**To Reproduce**
Steps to reproduce the behaviour:

Use Safari
1. Go to https://www.test.bbc.com/mundo (or any other home page with adverts turned on, on Test env)
2. Look at console

I only see the Safari error on front pages https://www.test.bbc.com/mundo not on other pages like most read https://www.test.bbc.com/mundo/popular/read or MAPs https://www.test.bbc.com/mundo/aprenda-ingles-23049220

This is unexpected, since the Content Security Policy Header is applied to all pages. The main difference is that we're loading adverts on home page.

When looking at the console for another home page without ads, e.g. https://www.test.bbc.com/afaanoromoo, there are no errors.

**Expected behaviour**
No errors

**Screenshots**
Screenshot of the mundo canonical home page in Safari - you can see the 
<img width="1294" alt="Screenshot mundo canonical home page csp error safari" src="https://user-images.githubusercontent.com/3028997/85428273-d43fb480-b574-11ea-8e26-c82f6a393369.png">

**Desktop (please complete the following information):**

- OS: Mac OS
- Browser Safari
- Version 12.1.2 (14607.3.9)

**Smartphone (please complete the following information):**
(Not tested on iPhone Safari - would be good to test to see the usecase)

**Testing notes**

- [x] This bug fix is expected to need manual testing.

**Additional context**

**Comment by DenisHdz** [comment]
The reason why we are seeing these errors on Safari is because it doesn't support the [worker-src](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/worker-src) and [report-to](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/report-to) directives.

**Comment by DenisHdz** [comment]
Blocked on fixing https://github.com/bbc/simorgh/issues/7214 so we can debug this locally and understand why these CSP erros just appear when the ads are on the page.

**Comment by DenisHdz** [comment]
In order to replicate this locally in Safari, we should set the `upgrade-insecure-requests` header to false as indicated in  https://github.com/bbc/simorgh/issues/7214#issuecomment-660137486

**Comment by mmcculloch** [comment]
The "refused to load" errors shown have been fixed by another issue. 

Safari doesn't support `worker-src` and `report-src`, these are supported by other browsers so we do not wish to remove them. We concluded that we should close this issue without change as this is likely a feature that will be added in future Safari versions.
